src/util.ts

```diff
@@ -0,0 +1,5 @@
+export type KeyPair = {
```

Comment:
This doesn't look like something that should be in `utils`. Also it should be `utils.ts`, not `util.ts`. This looks like a domain specific type, it should be in some relevant module.

Reply:
I've put it into the KeyManager class as an exported type